lib/smart_answer_flows/marriage-abroad/outcomes/countries/australia/_opposite_sex.govspeak.erb

```diff
@@ -2,7 +2,9 @@
 Read the [guidance for the state or territory](http://www.australia.gov.au/information-and-services/family-and-community/relationships/marriage-and-relationships) where you want to get married to find out what you need to do.
 ^ Check the [travel advice for Australia](/foreign-travel-advice/australia) and get [legal advice](/government/collections/list-of-lawyers) before making any plans.^
 
-##Prove you’re free to get married
+You might be able to form an opposite sex civil partnership. Check the local rules with the [British high commission](/world/organisations/british-high-commission-canberra).
+
+## Prove you’re free to get married
```

Comment:
Is the extra space an unintended side-effect? Not a problem though.

Reply:
It looks like some of the headings are missing the space so this just corrects it at the same time for consistency 😄